### PR TITLE
GDC blog post and event

### PR DESCRIPTION
**Pull request by coppolaemilio**
Wrote a blog post about the GDC event and more information about which games we are showcasing.

### CODE REVIEW

File: collections/_article/gdc-2023-godot-games.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+If you are interested in meeting us and other Godot developers outside of the GDC, on Wednesday, March 22 from 6:30 pm to 9:30 pm we are hosting an event at GitHub HQ open to anyone interested in Godot Engine: AAA, indie, and hobbyist developers of all levels are welcome. Snacks, drinks, talks, and Godot!
+
+Register for the event [here](https://lu.ma/2sz7890o).
+
+
+## Godot Games at our booth
+
+We understand that coming to the GDC is not something that everybody can afford, so we would like to give you the opportunity to showcase your game even if you are not able to attend. We prepared a form for you to [submit your game](https://forms.gle/ZVD1hR4sLr9nYMJC9) and we will select some of those to showcase at our booth.
+
+We are honored to be joined by the games and de developers of:
```
Comment (Calinou) [inline collections/_article/gdc-2023-godot-games.md]:
```suggestion
We are honored to be joined by the games and the developers of:
```

File: pages/events.html [hunk 2]
```diff
@@ -135,6 +140,30 @@ <h2 id="2023">2023</h2>
...
+				
+				<p>GitHub is generously hosting us again at their headquarters.</p>
+					
+				<p>​The event is open to anyone interested in Godot Engine: AAA, indie, and hobbyist developers of all levels are welcome. Snacks, drinks, talks, and Godot!</p>
+
+				<p>See the <a href="https://lu.ma/2sz7890o">event page</a> for details and registration.</p>
+
+				<ul>
+					<li><strong>Date:</strong> Wed, 22 March 2023, 6:30 pm to 9:30 pm</li>
+					<li><strong>Location:</strong> 275 Brannan St, San Francisco, CA 94107</li>
```
Comment (YuriSizov) [inline pages/events.html]: Missing OSM link 🙃

File: pages/events.html [hunk 3]
```diff
@@ -135,6 +140,30 @@ <h2 id="2023">2023</h2>
 		</div>
 
 		<div class="events-fulllist">
+			<a id="gdcmeetup-2023"></a>
+			<div class="card base-padding">
+				<h3>Godot GDC Meetup @ GitHub HQ</h3>
+
+				<img src="/storage/app/media/events/gdc-2023.jpg" alt="Godot GDC Meetup banner">
```
Comment (YuriSizov) [inline pages/events.html:147]: Can you also make a card for the GDC event itself?

File: pages/events.html [hunk 4]
```diff
@@ -135,6 +140,30 @@ <h2 id="2023">2023</h2>
...
+				
+				<p>Feels good to be back! To continue with our tradition, thanks to our friends over at GitHub we are hosting the third annual Godot GDC Meetup.</p>
```
Comment (YuriSizov) [inline pages/events.html]:
```suggestion
				<p>Feels good to be back! To continue with our tradition, thanks to our friends over at GitHub we are holding the third annual Godot GDC Meetup.</p>
```

So it doesn't conflict with the next line where GitHub is hosting.

File: collections/_article/gdc-2023-godot-games.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
+---
+title: "Godot & Friends at GDC 2023"
+excerpt: "We are going to be at the Games Developer Conference (GDC) 2023, which will take place in San Francisco from March 20th to March 24th."
```
Comment (YuriSizov) [inline collections/_article/gdc-2023-godot-games.md]:
```suggestion
excerpt: "We are going to be at the Game Developers Conference (GDC) 2023, which will take place in San Francisco from March 20th to March 24th."
```

File: collections/_article/gdc-2023-godot-games.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+		border-radius: 7px;
+		background-color: var(--card-background-color);
+		box-shadow: 0 5px 10px -3px #00000078;
+		display: block;
+		margin-bottom: 6px;
+	}
+</style>
+
+
+We are going to be at the Games Developer Conference (GDC) 2023, which will take place in San Francisco from March 20th to March 24th.
```
Comment (YuriSizov) [inline collections/_article/gdc-2023-godot-games.md]:
```suggestion
As we [announced at the end of the last year](/article/godot-at-gdc-2023/), Godot is going to be at the Game Developers Conference (GDC) 2023, which will take place in San Francisco from March 20th to March 24th.
```

File: collections/_article/gdc-2023-godot-games.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+
+At GDC 2023, we will have our own booth where we will be hosting game developers who will be showcasing their latest games.
```
Comment (YuriSizov) [inline collections/_article/gdc-2023-godot-games.md]:
```suggestion
For the first time, Godot is going to have its own booth at GDC where we will be hosting game developers who will be showcasing their latest games made with the engine.
```

Comment (YuriSizov) [reply collections/_article/gdc-2023-godot-games.md]: Though this sentence is just weird, as it kind of repeats both preceding and following sentiments. Can probably be rephrased and merged to flow better.

File: collections/_article/gdc-2023-godot-games.md [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+
+We would love to meet you! Come and chat with the team behind Godot, as well as the developers of some of the most popular games created with the engine. If you don't have a pass for GDC 2023, our sponsor W4 might be able to get one for you. Read more about it on [their blog post](https://w4games.com/2023/02/08/road-to-game-developers-conference-gdc/).
+
+## Godot Meetup
+
+If you are interested in meeting us and other Godot developers outside of the GDC, on Wednesday, March 22 from 6:30 pm to 9:30 pm we are hosting an event at GitHub HQ open to anyone interested in Godot Engine: AAA, indie, and hobbyist developers of all levels are welcome. Snacks, drinks, talks, and Godot!
```
Comment (YuriSizov) [inline collections/_article/gdc-2023-godot-games.md]:
```suggestion
If you are interested in meeting the team behind the engine, as well as other Godot developers, outside of the GDC show floor, there is going to be a less formal event as well! On Wednesday, March 22 from 6:30 PM to 9:30 PM we are holding an event at GitHub HQ open to anyone interested in Godot Engine: AAA, indie, and hobbyist developers of all levels are welcome. Snacks, drinks, talks, and Godot!
```

**Comment by coppolaemilio** [comment]
@YuriSizov thanks for all the feedback, I got to everything, so it should be ready to be merged :)

**Comment by YuriSizov** [comment]
That's not exactly what I meant by a banner for the GDC event, but I guess it's fine. I would rename the files though, so they don't conflict in future. So the GDC banner would be gdc-2023, or similar, and the github meetup can be github-meetup-2023, or similar.

**Review by YuriSizov — APPROVED** [review]

**Pull request merged**